layouts/shortcodes/rawhtml.html

```diff
@@ -0,0 +1,4 @@
+<!-- Author: Ana Ulin https://anaulin.org/blog/hugo-raw-html-shortcode/ -->
```

Comment:
I'm still new to this: is this attribution appropriate? I'm keen to give credit where credit is due.